content/tutorials/manage-the-rippled-server/configuration/configure-advisory-deletion.md

```diff
@@ -0,0 +1,96 @@
+# Configure Advisory Deletion
+
+In its default configuration, the `rippled` server automatically deletes outdated history of XRP Ledger state and transactions as new ledger versions become available. If your server uses most of its hardware resources during peak hours, you can configure the server to delete ledgers only when prompted by a command scheduled to run during off-peak hours to reduce the chances of online deletion
+
+## Prerequisites
+
+This tutorial assumes your server meets the following prerequisites:
+
+- You are on a supported operating system: Ubuntu Linux, Red Hat Enterprise Linux (RHEL), or CentOS.
+
+- The `rippled` server is already [installed](install-rippled.html) and [online deletion](online-deletion.html) is enabled.
+
+    The default configuration file enables online deletion after 2000 ledger versions.
+
+- A `cron` daemon is installed and running.
+
+    Ubuntu Linux runs a `cron` daemon by default.
+
+    On RHEL or CentOS, you can install the `cronie` package:
+
+        $ sudo yum install cronie
+
+- Your server has enough disk space to store your chosen amount of history in its ledger store.
+
+    See [Capacity Planning](capacity-planning.html) for details of how much storage is required for different configuration. With advisory deletion enabled, the maximum history a server may accumulate before deletion is equal to the number of ledger versions configured in the `online_delete` setting **plus** the amount of time between online deletion prompts.
+
+- You know which hours are least busy for your server.
+
+## Configuration Steps
+
+To configure advisory deletion with a daily schedule, perform the following steps:
+
+1. Enable `advisory_delete` in the `[node_db]` stanza of your `rippled`'s config file.
+
+        [node_db]
+        # Other settings unchanged ...
+      	online_delete=2000
+      	advisory_delete=1
+
+    - Set `advisory_delete` to `1` to require approval before running online deletion. (Set it to `0` to run online deletion automatically as new ledger versions become available.)
+    - Set `online_delete` to the minimum number of ledger versions to keep after running online deletion. The server accumulates more history than this until online deletion runs.
+
+    {% include '_snippets/conf-file-location.md' %}<!--_ -->
+
+2. Test running the [can_delete method][] to prompt the server to run online deletion.
+
+    You can use the [`rippled` commandline interface](get-started-with-the-rippled-api.html#commandline) to run this command. For example:
+
+        $ rippled --conf=/etc/opt/ripple/rippled.cfg can_delete now
+
+    The response indicates the maximum ledger index that may the server may delete from its ledger store. For example, the following message indicates that ledger versions up to and including ledger index 43633667 are to be deleted:
+
```

Comment:
are to be deleted once the latest ledger acquired is >= that value plus the value of online_delete.

Reply:
Added a note following the code example to clarify this.

Reply:
This sentence reads awkwardly to me. Instead of "are to be deleted", I think it should say "are available to be deleted", "may be deleted", or something like that.

Reply:
suggested edit:

>maximum ledger index that may the server may delete from its ledger store.

to

>maximum ledger index that the server may delete from its ledger store.